build/bp-messages/bp-messages-blocks.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * BP Messages Blocks Functions.
+ *
+ * @package   bp-blocks
+ * @subpackage \build\bp-messages\bp-messages-blocks
+ */
+
+namespace BP\Blocks;
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Register the messages component blocks.
+ *
+ * @since 9.0.0
+ */
+function register_messages_blocks() {
+	$blocks = array(
+		'bp/messages' => array(
```

Comment:
It should be the name of the block here 😄 
```suggestion
		'bp/sitewide-notices' => array(
```